zebrad/tests/acceptance.rs

```diff
@@ -20,6 +20,88 @@
 //! If it does not have any IPv4 interfaces, IPv4 localhost is not on `127.0.0.1`,
 //! or you have poor network connectivity,
 //! skip all the network tests by setting the `ZEBRA_SKIP_NETWORK_TESTS` environmental variable.
+//!
+//! ## Large/full sync tests
+//!
+//! This file has sync tests that are marked as ignored because they take too much time to run.
+//! Some of them require environment variables or directories to be present:
+//!
+//! - `FULL_SYNC_MAINNET_TIMEOUT_MINUTES` env variable: The total number of minutes we
+//! will allow this test to run or give up. Value for the Mainnet full sync tests.
+//! - `FULL_SYNC_TESTNET_TIMEOUT_MINUTES` env variable: The total number of minutes we
+//! will allow this test to run or give up. Value for the Testnet ful  sync tests.
+//! - `/zebrad-cache` directory: For some sync tests, this needs to be created in
+//! the file system, the created directory should have write permissions.
+//!
+//! Here are some examples on how to run each of the tests:
+//!
+//! ```console
+//! $ cargo test sync_large_checkpoints_mainnet -- --ignored --nocapture
+//!
+//! $ cargo test sync_large_checkpoints_mempool_mainnet -- --ignored --nocapture
+//!
+//! $ sudo mkdir /zebrad-cache
+//! $ sudo chmod 777 /zebrad-cache
+//! $ export FULL_SYNC_MAINNET_TIMEOUT_MINUTES=600
+//! $ cargo test full_sync_mainnet -- --ignored --nocapture
+//!
+//! $ sudo mkdir /zebrad-cache
+//! $ sudo chmod 777 /zebrad-cache
+//! $ export FULL_SYNC_TESTNET_TIMEOUT_MINUTES=600
+//! $ cargo test full_sync_testnet -- --ignored --nocapture
+//! ```
+//!
+//! Please refer to the documentation of each test for more information.
+//!
+//! ## Lightwalletd tests
+//!
+//! The lightwalletd software is an interface service that uses zebrad or zcashd RPC methods to serve wallets or other applications with blockchain content in an efficient manner.
+//! There are several versions of lightwalled in the form of different forks. The original
+//! repo is <https://github.com/zcash/lightwalletd> but Zecwallet Lite uses a custom fork: <https://github.com/adityapk00/lightwalletd>.
+//! The custom fork from adityapk00 is the one Zebra use for this tests:
```

Comment:
This might not be true after nu5. It seems we should change this before merging. We also need to open a PR to change https://github.com/ZcashFoundation/zebra/blob/main/book/src/user/lightwalletd.md#download-and-build-lightwalletd

```suggestion
//! The lightwalletd implementation Zebra use for this tests is the original one: <https://github.com/zcash/lightwalletd> 
```

Reply:
Zebra was changed to use the original fork in PR #4553. We guarantee that the original fork will work, but our tests should work with both.

We might want to keep the sentence that says there are multiple forks, because it is very confusing if you don't know that.